src/Resolver.js

```diff
@@ -185,4 +185,8 @@ export default class Resolver {
       return React.renderToStaticMarkup(context);
     });
   }
+
+  static decorate(props = {}) {
```

Comment:
Shouldn’t this include both `props` and `context` to not break decorated components requiring context?